app/datarequest/view.js

```diff
@@ -19,21 +19,23 @@ $(document).ready(function() {
     var datarequestUiSchema = {};
     var datarequestFormData = {};
 
-    // Get the schema, uiSchema and formData of the data request to be reviewed.
-    // Then render the data as a disabled form
-    axios.all([
-        axios.get("/datarequest/datarequest/schema"),
-        axios.get("/datarequest/datarequest/data/" + requestId)
-        ])
-        .then(axios.spread((schemaresponse, dataresponse) => {
-            datarequestFormData = dataresponse.data;
-            datarequestSchema   = schemaresponse.data.schema;
-            datarequestUiSchema = schemaresponse.data.uiSchema;
-
-            render(<ContainerReadonly schema={datarequestSchema} uiSchema={datarequestUiSchema} formData={datarequestFormData} />,
-                document.getElementById("datarequest")
-            );
-        }));
+    // Get data request
+    Yoda.call('datarequest_get',
+        {request_id: requestId},
+        {errorPrefix: "Could not get datarequest."})
+    .then((dr) => {
+        datarequestFormData = JSON.parse(dr.requestJSON);
+    });
+
+    // Get the schema and uiSchema, then render the data as a disabled form.
+    axios.get("/datarequest/datarequest/schema").then((schema) => {
```

Comment:
Is it possible to retrieve the schema, ui schema and data in one API call? That could save two requests.

Reply:
Out of scope of this PR IMO, but yes, that would be more efficient. Two problems:

1. The schemas are currently not retrieved from the back-end (to-do).
2. Bundling back-end calls could lead to an uncontrolled growth of "aggregate calls", i.e. for every view you would have to define an aggregate call in the back-end. Maybe we should first consider whether there's an elegant (generic) way to bundle calls?

Reply:
Ah yes, I forgot #1 is still a to-do.